src/components/PostList.tsx

```diff
@@ -1,9 +1,37 @@
 import React from "react";
+import { useAppSelector } from "../redux/hooks";
+import { FeedItem } from "../utilities/types";
+import { NoteActivityPub } from "../utilities/activityPubTypes";
 
 const PostList: React.FC = () => {
+  const feed: FeedItem[] = useAppSelector((state) => state.feed.feed);
+  feed.filter((post) => post?.content?.type === "Note");
+
+  const postItems = feed.map((post: FeedItem, index: number) => {
```

Comment:
I think we can refactor the individual postItem into a their own component.
```
const PostItem = ({fromAddress, noteContent, attachement, ....}) => (
  <div>
  </div>
);
```

Reply:
this is happening in another story.  the current code in postlist is just a placeholder.